src/alcotest/alcotest.mli

```diff
@@ -111,6 +111,9 @@ val reject : 'a testable
 val check : 'a testable -> string -> 'a -> 'a -> unit
 (** Check that two values are equal. *)
 
+val check_l : t:'a testable -> msg:string -> expected:'a -> actual:'a -> unit
```

Comment:
```suggestion
val check' : t:'a testable -> msg:string -> expected:'a -> actual:'a -> unit
```
I'd normally expect such "syntax alternatives with equivalent behaviour" to be primed: slightly less descriptive, slightly easier to type. What do you think? (Perhaps @samoht has an opinion.)

Reply:
i like the primed check function too :)